lms/static/scripts/frontend_apps/components/RelativeTime.tsx

```diff
@@ -8,17 +8,29 @@ import { useEffect, useMemo, useState } from 'preact/hooks';
 export type RelativeTimeProps = {
   /** The reference date-time, in ISO format */
   dateTime: string;
+
+  /**
+   * Whether a title with the absolute date should be added.
```

Comment:
```suggestion
   * Whether a `title` attribute with the absolute date should be added.
```

Clarify this refers to an HTML `title` rather than a visible string.